osu.Game/Overlays/Chat/ChatTextBox.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using osu.Framework.Bindables;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Overlays.Chat
+{
+    public class ChatTextBox : FocusedTextBox
```

Comment:
I'll note that on design this chat box's placeholder text should have `Background1` colour rather than the `Foreground1` that it's currently using from base - not that I *hate* the current look, mind you...